beaker/client/application_client.py

```diff
@@ -38,6 +38,7 @@ def __init__(
         self.client = client
         self.app = app
         self.app_id = app_id
+        self.app_addr = None
```

Comment:
can you make this one a check for if app_id is set, if it is set the address to the `get_application_address(app_id)`

Reply:
Does this look better?